testsuite/MDAnalysisTests/analysis/test_base.py

```diff
@@ -139,12 +137,9 @@ def simple_function(mobile):
 def test_AnalysisFromFunction():
     u = mda.Universe(PSF, DCD)
     step = 2
-    ana1 = base.AnalysisFromFunction(simple_function, mobile=u.atoms,
-                                     step=step).run()
-    ana2 = base.AnalysisFromFunction(simple_function, u.atoms,
-                                     step=step).run()
-    ana3 = base.AnalysisFromFunction(simple_function, u.trajectory, u.atoms,
-                                     step=step).run()
+    ana1 = base.AnalysisFromFunction(simple_function, mobile=u.atoms, step=step).run()
```

Comment:
please don't change this formatting. I'm sure this was done to conform to the 80 char limit of pep8